### PR TITLE
coalesce HTML refs

### LINKED ISSUE #996: coalesce HTML references?
presently our editors' draft references three HTML specs:
>[HTML]
    Anne van Kesteren; et al. HTML Standard. Living Standard. URL: https://html.spec.whatwg.org/multipage/ 
[HTML52]
    Steve Faulkner; et al. HTML 5.2. 14 December 2017. REC. URL: https://www.w3.org/TR/html52/ 
[HTML53]
    HTML 5.3. 14 December 2017. WD. URL: https://www.w3.org/TR/html53/ 

Might we be able to reference just one?  Unless there's something "new" only in [HTML] that we need to specifically reference?

cc: @wseltzer @samweiler

### PR DESCRIPTION
**Pull request by equalsJeffH**
fix #996

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1243.html" title="Last updated on Jun 27, 2019, 12:03 AM UTC (0e6495c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1243/78ed130...0e6495c.html" title="Last updated on Jun 27, 2019, 12:03 AM UTC (0e6495c)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -122,7 +117,7 @@ spec: ECMAScript; urlPrefix: https://tc39.github.io/ecma262/#
 spec: RFC8152; urlPrefix: https://tools.ietf.org/html/rfc8152
     type: dfn
         text: Section 7; url: section-7
-
+<!--
```
Comment (emlun) [inline index.bs]: Why not delete the obsolete reference?

File: index.bs [hunk 2]
```diff
@@ -132,6 +127,22 @@ spec: HTML53; urlPrefix: https://w3c.github.io/html/
...
+        urlPrefix: dom.html
+            for: Document; url: concept-document-feature-policy; text: feature policy
+        urlPrefix: browsers.html
+            text: browsing context; url: browsing-context
+            text: origin; url: concept-origin
+            text: opaque origin; url: concept-origin-opaque
+            text: tuple origin; url: concept-origin-tuple
+            text: document.domain; url:dom-document-domain
+            text: scheme; url: origin-scheme
+            text: port; url: origin-port
```
Comment (emlun) [inline index.bs]: Some links are broken, looks like they should be:

- `origin`: `urlPrefix: origin.html`
- `opaque origin`: `urlPrefix: origin.html`
- `tuple origin`: `urlPrefix: origin.html`
- `scheme`: https://url.spec.whatwg.org/#concept-url-scheme
- `port`: https://url.spec.whatwg.org/#concept-url-port

File: index.bs (hunk 1 above)
Comment (equalsJeffH) [reply index.bs]: oh sorry, fergot it was still lying about...   deleted.

File: index.bs (hunk 2 above)
Comment (equalsJeffH) [reply index.bs]: are you building locally? if so, have you done ` bikeshed update` recently?  It builds without errors for me on https://api.csswg.org/bikeshed/

Comment (emlun) [reply index.bs]: Yeah, it builds without errors, but the links don't lead where they're supposed to. For example, [opaque origin](https://html.spec.whatwg.org/multipage/browsers.html#concept-origin-opaque) should be https://html.spec.whatwg.org/multipage/origin.html#concept-origin-opaque

Comment (equalsJeffH) [reply index.bs]: doh.  ok, thanks for checking the links. all worked (except scheme and port) for me, apparently due to some redirect magic on the part of the whatwg html spec, but probably better to have 'em not relying on that...  so I've updated those dfn links. Does it look OK now?

Comment (emlun) [reply index.bs]: Oh, I see, there's some JavaScript redirect that was blocked by my NoScript. :smile:

Yes, it's working now!

**Pull request merged**